bartender/filesystems/local.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from bartender.filesystems.abstract import AbstractFileSystem
+from bartender.schedulers.abstract import JobDescription
+
+
+class LocalFileSystem(AbstractFileSystem):
+    """File system operations on file system current Python process is running on."""
+
+    def localize_description(
+        self,
+        description: JobDescription,
+        entry: Path,
+    ) -> JobDescription:
+        """Make given job description local to this file system.
+
+        :param description: A job description.
+        :param entry: The path to replace with the entry path of this file system.
+            For example given a file system with entry path of /remote/jobs
+            and given job_dir in job description of /local/jobs/myjobid
+            and given entry of /local/jobs
+            will return description with job dir /remote/jobs/myjobid .
+        :return: A job description local to this file system.
+        """
+        # TODO do something with entry argument?
+        return description
```

Comment:
Is this also intended to "translate" the path from a remote jobdescription to a corresponding jobdescription on the local fs?

Reply:
Postponed to https://github.com/i-VRESSE/bartender/issues/34